<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customsightseeing</class>
 <widget class="QDialog" name="customsightseeing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#customsightseeing {
  border-image:url(:/images/images/background.PNG);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#002244;</string>
     </property>
     <property name="text">
      <string>Select Sightseeing Activitites (Maximum Five)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Shangri-La Lake: Jeep safari to a pristine glacial lake.  ----PKR 7500 (including food)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Kargah Buddha: See a 7th-century rock-carved statue.   -----PKR 100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Altit &amp; Duikar Forts: Explore  Altit Fort and admire the views.   ----PKR 900 (including guide)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Naltar Valley: Ride horses or relax in the scenery.  ----PKR 5000 (including horse rides and food)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Karakoram Highway: Wind through mountain passes on this world's highest road  ---PKR 15000 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Fairy Meadows: Hike to meadows and see Nanga Parbat.  ---PKR 750</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Hunza Valley: Journey through the picturesque Hunza Valley  ---PKR 5000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextbtn">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 10px;
border-radius:10px;
background-color: rgb(89, 170, 163);
color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
